frontend/src/ScoreTable.js

```diff
@@ -1,87 +1,61 @@
-import React, { Component } from 'react'
+import React, { useEffect, useState } from 'react'
 import ReactTable from 'react-table'
 
-/**
- * A wrapper for the react table with common table functionality and styling.
- */
-class ScoreTable extends Component {
-  constructor(props) {
-    super(props)
+const ScoreTable = props => {
+  const { noun, multi, data = [], onSelect = () => {} } = props
```

Comment:
can use object destructuring when passing in props, so instead of saying `props =>`, u can say `({ noun, multi, data = [], onSelect=()=>{} }) => `

Reply:
can't, I need the entire props object to spread into the ReactTable component below